editor.planx.uk/src/@planx/components/List/schemas/GLA/ExistingAndProposedUses.ts

```diff
@@ -0,0 +1,56 @@
+import { Schema } from "@planx/components/List/model";
+
+export const ExistingAndProposedUsesGLA: Schema = {
+  type: "Existing and proposed uses",
+  fields: [
+    {
+      type: "question",
+      data: {
+        title: "What is the use class?",
+        fn: "useClass",
+        options: [
+          { id: "bTwo", data: { text: "B2 - General industry", val: "bTwo" } },
+          { id: "bEight", data: { text: "B8 - Storage and distribution", val: "bEight" } },
+          { id: "cOne", data: { text: "C1 - Hotels", val: "cOne" } },
+          { id: "cTwo", data: { text: "C2 - Residential institutions", val: "cTwo" } },
+          { id: "cTwoA", data: { text: "C2a - Secure residential institutions", val: "cTwoA" } },
+          { id: "cThree", data: { text: "C3 - Dwellinghouses", val: "cThree" } },
+          { id: "cFour", data: { text: "C4 - Houses in multiple occupation", val: "cFour" } },
+          { id: "e", data: { text: "E - Commercial, business and service", val: "e" } },
+          { id: "fOne", data: { text: "F1 - Learning and non-residential institutions", val: "fOne" } },
+          { id: "fTwo", data: { text: "F2 - Local community uses", val: "fTwo" } },
+          { id: "SG", data: { text: "Sui generis", val: "SG" } },
+          { id: "unknown", data: { text: "Unknown", val: "unknown" } },
+        ],
+      },
+    },
+    {
+      type: "number",
+      data: {
+        title: "What is the existing gross internal floor area?",
+        units: "m²",
+        fn: "area.existing",
```

Comment:
I suspect these `.` separated `fn` are the source of your bug! 

For now, I think we need to write all `fn` & `val` _without_ separators - but I'll work on finding a fix to support this / try to wrap my head around why Daf may not have done it from the start ! 

Feel free to note "ideal" `fn` in comment at line ends like this so we don't forget: 
```
fn: "areaExisting", // area.existing
```